modules/mixed-arch-import-imagestreams.adoc

```diff
@@ -0,0 +1,26 @@
+//Module included in the following assemblies
+//
+//post_installation_configuration/cluster-tasks.adoc
+
+:_content-type: PROCEDURE
+[id="mixed-arch-import-imagestreams_{context}"]
+
+= Importing manifest lists on your heterogeneous cluster 
+
```

Comment:
Hmm...maybe a better title would be:

"Importing manifestlisted images in imagestreams on your hetrogeneous cluster"

or something to that effect

Reply:
also manifestlist is one word...not separate...but could you check how that is in other docs? we must follow the same convention

Reply:
Hmm, in the docs it looks like we use manifest lists. But its not in all of our tracking, and its not a common term used yet in the docs as well, so let me check with some of the other writers on this

Reply:
and the same with image streams as well..i think it is just imagestreams...

Reply:
It looks like the images writer used them as separate in his [RN PR](https://github.com/openshift/openshift-docs/pull/51191/), but I'll ask him as well just in case!

Reply:
The IR writer said that we use `manifest list` and `image streams`